Snowflake.Data.Tests/UnitTests/SFDbParameterTest.cs

```diff
@@ -125,5 +127,75 @@ public void TestDbParameterResetDbType([Values] SFDataType expectedSFDataType)
             _parameter.ResetDbType();
             Assert.AreEqual(SFDataType.None, _parameter.SFDataType);
         }
+
+        [Test]
+        public void TestDbTypeExplicitAssignment([Values] DbType expectedDbType)
```

Comment:
what if DbType is not provided?
I would suggest to have a special edge case tests for:
- null value and (just one) non null DbType
- non null value and null DbType
- both - DbType and Value nulls

Reply:
I added a test case for the first item on the list. And I also added a test case for null value and default DbType

But I'm not able to add a test case for the other two because `DbType` is a non-nullable value type. It always defaults to `DbType.AnsiString`

Reply:
Thanks @sfc-gh-ext-simba-lf !